zookeeper-server/src/test/java/org/apache/zookeeper/common/ZKTrustManagerTest.java

```diff
@@ -85,20 +106,7 @@ public static void removeBouncyCastleProvider() throws Exception {
     public void setup() throws Exception {
         mockX509ExtendedTrustManager = mock(X509ExtendedTrustManager.class);
 
-        mockInetAddress = mock(InetAddress.class);
```

Comment:
We are still left with a `mockInetAddress` member variable, which could cause some confusion since it's not really a mock anymore.

Can the member variable be removed completely? (Maybe not. Maybe it's still needed to return from the mock socket.)

If we still need it, then I suggest renaming it to `inetAddress` so that it's clearer that it's not really a mock.

Reply:
we don't need it anymore, removed